tests/trace_plugin_test.py

```diff
@@ -95,6 +96,20 @@ def test_TraceApi(self) :
         global testSuccessful
         testSuccessful = True
 
+        # relaunch with no time allocated for http response & abi-serializer. Will fail because get_info fails.
+        node.kill(signal.SIGTERM)
+        Utils.Print("Ignore expected: ERROR: Node relaunch Failed")
+        isRelaunchSuccess = node.relaunch(timeout=10, addSwapFlags={"--http-max-response-time-ms": "0", "--abi-serializer-max-time-ms": "0"})
```

Comment:
Will the relaunch fail as the message indicates?
Also wondering if it is useful to assign to `isRelaunchSuccess`?

Reply:
`get_info` can't fail in `main`, so it actually does work. I'll update the test.